stories/tooltip/generator.tsx

```diff
@@ -32,7 +32,7 @@ const generateStoryFunction = ({ position, alignment }) => () => {
               gravida.
             </p>
           )}
-          getPosition={() => position}
+          preferredPosition={position}
```

Comment:
why is it `preferred` and not just the absolute position? will it pop down when there's no space to pop up even though I preferred it to go up?

Reply:
As the docs state: https://github.com/carforyou/carforyou-components-pkg/pull/220/files#diff-4f6c0f244f5e6da59ac54389842bea08R32-R34

So yes, you're correct - if we would want to render tooltip on the top but there's not enough space it will be rendered on the bottom instead.

Reply:
I opted for adding `preferred` prefix to signify that. Happy to switch back to `position` like most of the tooltips do